trellis/trellis.go

```diff
@@ -129,6 +129,35 @@ func (t *Trellis) SiteNamesFromEnvironment(environment string) []string {
 	return names
 }
 
+func (t *Trellis) FindSiteNameFromEnvironment(environment string, siteNameArg string) (string, error) {
```

Comment:
Should we unit test this?

Reply:
Added along with some other `*Trellis` unit tests

Reply:
Not even close 😞 but in my defense a CLI like this makes it hard sometimes